pkg/binding/asker/survey_asker.go

```diff
@@ -6,6 +6,11 @@ import (
 	"github.com/AlecAivazis/survey/v2"
 )
 
+const (
+	BindAsFiles  = "Bind As Files"
+	BindAsEnvVar = "Bind As Environment Variables"
+)
```

Comment:
I find creating a type before `const` declaration to be more helpful since it enables us to use `switch..case` statement unlike `if..else`. It's not an issue here, but when we have more items/possibilities, it comes in handy. Something like:

```go
type BindAs string

const (
	BindAsFiles  BindAs = "Bind As Files"
	BindAsEnvVar BindAs = "Bind As Environment Variables"
)
```
(Not a request to change anything; just sharing my understanding. I'm using it in my upcoming `odo logs` to switch between `--dev`, `--deploy`, none)

Reply:
I'll keep this in mind, but for now, I do not think we need to follow this convention.